src/main/java/com/backbase/ct/bbfuel/configurator/PocketsConfigurator.java

```diff
@@ -96,17 +101,20 @@ public String ingestPocketArrangementForModeOnetoOneAndSetEntitlements(
      * Ingests Pockets for the given user.
      *
      * @param externalUserId External user ID to ingest pockets for.
+     * @return List<Pocket>
      */
-    public void ingestPockets(String externalUserId) {
+    public List<Pocket> ingestPockets(String externalUserId) {
         log.debug("Going to ingest pockets for user [{}]", externalUserId);
-
+        List<Pocket> createdPockets = Collections.synchronizedList(new ArrayList<>());
         List<PocketPostRequest> pockets = pocketsReader.load();
 
         for (PocketPostRequest pocketPostRequest : pockets) {
             Pocket pocket = pocketsRestClient.ingestPocket(pocketPostRequest);
             log.info("Pocket with ID [{}] and name [{}] created for user [{}]", pocket.getId(), pocket.getName(),
                 externalUserId);
+            createdPockets.add(pocket);
         }
+        return ListUtils.unmodifiableList(createdPockets);
```

Comment:
Can be replaced with `Collections.unmodifiableList(createdPockets)`.

Reply:
changed